### ISSUE TITLE
I tried running this script but this is is all i got... wtf

### ISSUE THREAD

**Issue by wayjake** [dannydi12/recall#1]
![image](https://github.com/user-attachments/assets/cbdd150e-a24d-49be-bd50-1d77becb6bad)

**Comment by dannydi12** [comment]
Yeah, this project is still in early alpha. So it can result in unexpected behavior like this. Thank you for reporting!

**Comment by wayjake** [comment]
Dear Daniel,

I noticed some concerning security practices in your public repository. From a security best practices standpoint, this level of visibility could pose _potential risks_.

**I strongly recommend either:**
Making the repository private, or
Removing it entirely

This isn't meant to be intrusive, but rather a professional courtesy from one developer to another regarding security best practices.

Best regards,
Jake

P.S. Since you mentioned this project is in early alpha, this would be an ideal time to implement these security changes.
P.P.S. Since you pinned this, it signales to me that you are willing to take this issue seriously; though i still disagree with this being left open to the pulic for (LiteralLY anyone) to see, i commend you for you semblance of respect towards the OPENSSSOURECSSED (OSS i think it is for shorrt, i understand your underdeveloped brain may not be able to remember that many words (chars) in an accronym) community. 

Sent from my iPhone
WARNING: This product can expose you to chemicals including [Chemical Names], which are known to the State of California to cause cancer and birth defects or other reproductive harm. For more information, go to [URL].

**Comment by wayjake** [comment]
actually though.. on debian i had to install this to get it running: https://github.com/libvips/pyvips/issues/31#issuecomment-367713852

at first i didn't include the -dev portion which led me here: https://github.com/puppeteer/puppeteer/issues/1598. 
but i don't think the flag was needed. 

then this: https://github.com/cypress-io/cypress/issues/4631#issuecomment-507717436

then this: https://stackoverflow.com/questions/67407104/error-while-loading-shared-libraries-libgbm-so-1-cannot-open-shared-object-fil

and then finally this: https://gist.github.com/tavinus/7effd4b3dac87cb4366e3767679a192f#chromium-dependencies

ANYWAY, IT STILL DIDN'T WORK

![image](https://github.com/user-attachments/assets/7255e756-1238-422c-a087-cceda165419e)

**Comment by wayjake** [comment]
THEN I JUST ADhD A NEW DEBIAN USER

## Step 1: Log in as root
sudo su -

## Step 2: Create a new user
### Replace 'newusername' with your desired username
adduser newusername

## Step 3: Add the user to the sudo group (optional, for admin privileges)
usermod -aG sudo newusername

## Step 4: Log in as the new user
### Exit the root session
exit

## Switch to the new user
su - newusername

## Alternatively, log in via SSH (if applicable)
### Replace 'your-server-ip' with your server's IP address
ssh newusername@your-server-ip

**Comment by wayjake** [comment]
EVERYTHING IS FINE. REMAIN F'ING CALM
![image](https://github.com/user-attachments/assets/249847d0-c7ed-402b-ac54-86bb510c71bd)

**Comment by wayjake** [comment]
honestly, i kinda forgot this is a native app so i've been trying to run it in the cloud for no reason. i have fun anyway https://forums.debian.net/viewtopic.php?t=12256

**Comment by wayjake** [comment]
lmao --- on my mac:

yarn
yarn start

![image](https://github.com/user-attachments/assets/131b834e-1b15-4e0b-a31a-45b0d356c374)
